### PR TITLE
CMR-6795 - Making initial changes from feedback

### PR DESCRIPTION
**Pull request by jceaser**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
+3. Test the changes locally/in your personal fork.
+4. Submit a pull request to open a discussion about your proposed changes.
+5. The maintainers will talk with you about it and decide to merge or request additional changes.
+
+## Commits
+
+Our ticketing and CI/CD tools are configured to sync statuses amongst each
+other. Commits play an important role in this process. Please start all commits
+with the CMR ticket number associated with your feature, task, or bug. All
+commit messages should follow the format "CMR-XXXX: [Your commit message here]"
```
Comment (jaybarra) [inline CONTRIBUTING.md:20]: Outside contributors won't have knowledge of a specific ticket number.

Comment (jceaser) [reply CONTRIBUTING.md]: This was also pointed out on slack, I have made a change

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+2. Create a branch and make your changes, including tests to verify your change
+3. Test the changes locally/in your personal fork.
+4. Submit a pull request to open a discussion about your proposed changes.
+5. The maintainers will talk with you about it and decide to merge or request additional changes.
+
+## Commits
+
+Our ticketing and CI/CD tools are configured to sync statuses amongst each
+other. Commits play an important role in this process. Please start all commits
+with the CMR ticket number associated with your feature, task, or bug. All
```
Comment (tbs1979) [inline CONTRIBUTING.md:20]: How would end-users know about CMR ticket numbers?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,7 +1,31 @@
...
+commit messages should follow the format "CMR-XXXX: [Your commit message here]"
+
+## Disclaimer
+
+The Earthdata Search develoment team will review all pull requests submitted.
```
Comment (tbs1979) [inline CONTRIBUTING.md]: don't you mean CMR team? and you have a misspelling

**Review by tbs1979 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -27,10 +27,12 @@ summery on naming conventions can be found [here][pnames].
 
 ## Disclaimer
 
-The CMR develoment team will review all pull requests submitted.
-Only requests that meet the standard of quality set forth by existing code,
-following the patterns set forth by existing code, and adhering to the design
-patterns set forth by existing UI elements will be considered and/or accepted.
+For now, the CMR develoment team will be taking a lead on review all pull
```
Comment (tbs1979) [inline CONTRIBUTING.md:30]: to

File: README.md [hunk 5]
```diff
@@ -1 +1,9 @@
-This repository is for an upcoming project to provid tools for interfacing with NASA metadata systems.
+# EO Metadata Tools #
+
+This repository is for an upcoming project to provid tools for interfacing with
```
Comment (tbs1979) [inline README.md:3]: misspelling....should be provide

File: README.md [hunk 6]
```diff
@@ -1 +1,9 @@
...
+NASA metadata systems. Each directory possibly written in different programing
```
Comment (tbs1979) [inline README.md:4]: misspelling....should be programming

File: CONTRIBUTING.md (hunk 2 above)
Comment (jceaser) [reply CONTRIBUTING.md:20]: I'll fix that, it has been changed to github

File: CONTRIBUTING.md (hunk 3 above)
Comment (jceaser) [reply CONTRIBUTING.md]: your looking at an old copy

File: README.md (hunk 6 above)
Comment (jceaser) [reply README.md:4]: my spell checked does not agree. provid however was wrong and I fixed that

**Review by tbs1979 — APPROVED** [review]

**Review by jaybarra — APPROVED** [review]

**Pull request merged**